lib/devices/builtins/thingengine.builtin.js

```diff
@@ -51,6 +51,8 @@ module.exports = class MiscellaneousDevice extends Tp.BaseDevice {
         return [{ time: new Date }];
     }
     get_get_random_between({ low, high }) {
+        low = low || 1;
+        high = high || 6;
```

Comment:
You should check for null/undefined explicitly here, otherwise one cannot ask for "a random number between 0 and ...", because 0 is false.